### ISSUE TITLE
Missing thread notifications

### ISSUE THREAD

**Issue by bnjbvr** [element-hq/element-web#27519]
### Steps to reproduce

Open a thread and get responses. Alternatively, get mentioned in a thread you haven't started.

#### What did you expect?

TAC shows notifications for those threads.

#### What happened instead?

TAC didn't show notifications for those threads.

---

See rageshake: https://github.com/matrix-org/element-web-rageshakes/issues/26078

### Operating system

btw I use Arch

### Browser information

Firefox Developer Edition 127b9

### URL for webapp

https://chat.element.io

### Will you send logs?

Yes

**Comment by dbkr** [comment]
I think this is describing two different bugs:
 * Non-mention responses in a thread you started appearing in the TAC
 * Notifications for regular (non-mention) thread responses appearing in the TAC

**Comment by bnjbvr** [comment]
Not quite: messages *didn't* appear in the TAC. I didn't have any notification at all in the TAC. I expected notifications for:

- message answers (be they mention or not) for threads I have started
- mentions for any kind of threads

And none of these showed up.
